packages/theming/atlas/src/themesource/atlas_ui_resources/native/core/widgets/introscreen.ts

```diff
@@ -1,7 +1,7 @@
 import { isIphoneWithNotch } from "../helpers/_functions/device";
 import { font, spacing } from "../variables";
 import { IntroScreenButtonType, IntroScreenType } from "../../types/widgets";
-import { introScreen } from "../../app/custom-variables";
+import { introScreen } from "../../../../../theme/native/custom-variables";
```

Comment:
or should we always import `core/variables` from all files here since it already imports custom-variables?

Reply:
Yes you are right, fixed!